### DISCUSSION TITLE
How to test my Translation Efforts?

### DISCUSSION THREAD

**Discussion by mazetar** [bafolts/terraforming-mars#3900] [Localization]
Hi,

I'm currently making good progress on a rough translation for Norwegian.
But I'm struggling with testing my progress.

Currently I've cloned the repo to my Github, done my changes locally by creating a no folder based of the cn folder.
Deleted the files that I haven't translated from the NO folder. (Since I saw most translations didn't have all the files).

But Norwegian does not show in the list of languages to choose from. 
I assume I could cheat and just delete the cn folder and call mine cn to test, but is there a simple way to add a localization to the project?

**Comment by mazetar** [comment]
I've started testing by naming the folder CN and started correcting some issues.
If I'm able to get a Norwegian copy of the board game then I intend to go back and change my word choices to match the official board game's wording.

Gonna do some playtesting today and tomorrow and if all goes well I should be able to do a Pull Request by then :)

**Reply by kberg** [reply]
@mazetar what is going on with this effort? Are you still interested in adding the language?

**Comment by kberg** [comment]
Does editing `src/constants.ts` work?

![image](https://user-images.githubusercontent.com/413481/147846044-ce0983c3-eb70-4220-8cbe-b842bbd07149.png)
